### DISCUSSION TITLE
생성한 캐릭터의 프로필을 보이지 않게 할 수 없나요?

### DISCUSSION THREAD

**Discussion by earthboom** [naverz/zepeto-studio-kor#1032] [5. Q&A]
안녕하세요.

NPC로 사용하기 위해, 생성한 캐릭터를 터치하면 해당 ID의 프로핊이 보이게 됩니다.

이를 스크립트를 통해 컨트롤할 수 없나 하여 문의를 남깁니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, ZepetoPlayers > FloatingUI(onApp) > MiniProfile 설정으로 on/off 제어가 가능합니다.
다만 현재로서는 스크립트를 통한 제어는 불가합니다.

**Reply by earthboom** [reply]
안녕하세요.
답변 감사드립니다.
해당 설정은 전체적으로 프로필이 보이지 않는 것 같은데, 혹시 '생성한 캐릭터'만 한정하여 컨트롤은 아직 불가능한 건 가요?
유저들은 프로필이 보이게 하고 싶어서요.

**Reply by Zepeto-TechEvangelist** [reply]
네 현재는 불가합니다.
하지만 최근에 Floating UI 관련 동적 제어 니즈가 많아진 것을 감안하여 
지원하는 방향으로 검토해 보겠습니다.
감사합니다.
